app/code/Magento/Inventory/Controller/Adminhtml/Stock/InlineEdit.php

```diff
@@ -72,6 +73,13 @@ public function execute()
                         '[ID: %value] The Stock does not exist.',
                         ['value' => $itemData[StockInterface::STOCK_ID]]
                     );
+                } catch (ValidationException $e) {
```

Comment:
Also need to do same for \Magento\Inventory\Controller\Adminhtml\Source\InlineEdit

Reply:
I'm working on it, thanks.